components/events/Events.jsx

```diff
@@ -0,0 +1,30 @@
+import Content from "../Content";
+import GridEvents from './GridEvents.jsx';
+import EventComp from './Event.jsx';
+import A from "../Anchor";
+
+export default class Events extends React.Component{
+
+  render() {
+    const { posts } = this.props;
+    const { urlApi } = this.props;
+    return (
+      <section id="eventos">
+        <Content className="center mw6 tc pv5" component="article">
+          <h1>Eventos</h1>
+          <p>
+            Los realizamos una vez por mes, durante 2 horas y compartimos algunos
+            snacks. Aunque la entrada es gratuira es necesario que reserves.
```

Comment:
typo on `gratuira`

Reply:
Jajaja. Fine, but that typo is comming from your last pull request.  :P
but don't worry, I'll fix that